pkg/log/log.go

```diff
@@ -10,4 +10,7 @@ var (
 	Errorf = func(message string, obj ...interface{}) {
 		log.Printf("ERROR: "+message+"\n", obj...)
 	}
+	Debugf = func(message string, obj ...interface{}) {
+		log.Printf("DEBUG: "+message+"\n", obj...)
+	}
```

Comment:
nit:  Shouldn't these just be functions instead of variables pointing to functions?
```suggestion
)
func Debugf(message string, obj ...interface{}) {
	log.Printf("DEBUG: "+message+"\n", obj...)
}
```

Reply:
I did not write the package and am not sure what the intention was. For now I am conforming to how the other logs forms are written but for the longer term I made this issue for replacing it with an external logging package: https://github.com/rancher/wrangler/issues/244.